ntoskrnl/mm/ARM3/expool.c

```diff
@@ -2552,6 +2587,10 @@ ExFreePoolWithTag(IN PVOID P,
             Tag &= ~PROTECTED_POOL;
         }
 
+#if DBG
+        RtlFillMemory(P, PageCount * PAGE_SIZE, 0xCC);
```

Comment:
`0xCC` is also used in msvc debug builds as 'uninitialized var'
How about using something like `0xCD` or `0xDD` to indicate that this memory has been free'd instead?

Reply:
What about filling it with 0xDEADBEEF ULONGs?

Reply:
Works for me,
but keep in mind that `0xDEADBEEF` is also used in f.e. winetests a lot, so you'd have to track down first that it didnt come from userland.

Reply:
I switched to 0xCD for uninitialized and 0xDD for freed pool to match what MSVC does with the heap on debug builds.